### PR TITLE
Course Comparison 

### PR DESCRIPTION
**Pull request by Eric-R-Liang**
- Added course comparison feature --> display relevant course information based on user selections 
- Modified css files in frontend/build

### CODE REVIEW

**Comment by EmmaZhao0413** [comment]
please remove the generated files and push again. Otherwise the files changed are too long to review

**Comment by EmmaZhao0413** [comment]
> please remove the generated files and push again. Otherwise the files changed are too long to review

i.e. remove node_module folder, or checkout main

**Comment by Eric-R-Liang** [comment]
> please remove the generated files and push again. Otherwise the files changed are too long to review

Thanks for the review. I'll delete the files and re-push.

**Comment by Eric-R-Liang** [comment]
> > please remove the generated files and push again. Otherwise the files changed are too long to review
> 
> Thanks for the review. I'll delete the files and re-push.

Deleted node modules and re-pushed

File: Education_Pathways/__init__.py [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+			year-=1
+	return tables
+
+
+#def minor_results(minor_choice):
+	
+
+
+
+#with open('resources/course_vectorizer.pickle','rb') as f:
```
Comment (coelhona) [inline Education_Pathways/__init__.py]: Should delete commented blocks of code if they are not necessary

**Review by coelhona — COMMENTED** [review]
Please add comments regarding the major functions and changes you have made. Other than that the code functionality looks good to me, after adding unit tests and comments it should be good to go.

File: Education_Pathways/__init__.py (hunk 1 above)
Comment (Eric-R-Liang) [reply Education_Pathways/__init__.py]: Thanks. Got it

**Comment by Eric-R-Liang** [comment]
Sounds good. Will do

**Pull request merged**